src/pages/workspace/tags/WorkspaceTagsSettingsPage.tsx

```diff
@@ -39,32 +48,25 @@ function WorkspaceTagsSettingsPage({route, policyTags}: WorkspaceTagsSettingsPag
     const isLoading = !PolicyUtils.getTagLists(policyTags)?.[0] || Object.keys(policyTags ?? {})[0] === 'undefined';
     const {isOffline} = useNetwork();
     const hasEnabledOptions = OptionsListUtils.hasEnabledOptions(Object.values(policyTags ?? {}).flatMap(({tags}) => Object.values(tags)));
+    const {canUseWorkspaceRules} = usePermissions();
+
     const updateWorkspaceRequiresTag = useCallback(
         (value: boolean) => {
             Tag.setPolicyRequiresTag(policyID, value);
         },
         [policyID],
     );
 
+    function toggleBillableExpenses(policy: OnyxEntry<OnyxTypes.Policy>) {
```

Comment:
I'm pretty sure we can even extract it outside the component, since `policyID` that it's using is the same as `policy.id` from already used `policy` argument